openpype/hosts/resolve/api/plugin.py

```diff
@@ -420,8 +421,8 @@ def update(self, timeline_item):
         # create clip media
         media_pool_item = lib.create_media_pool_item(
             self.data["path"], self.active_bin)
-        clip_property = media_pool_item.GetClipProperty()
-        clip_name = clip_property["File Name"]
+        _clip_property = media_pool_item.GetClipProperty
+        clip_name = _clip_property("File Name")
```

Comment:
Unless there was an optimization in V17 you will generally get faster response if you cherry pick the property of interest right away instead of getting all clip properties: `clip_property = media_pool_item.GetClipProperty('File name')`

Reply:
Yes, I have found out that in v17 None is returned if the GetClipProperty is on an audio clip and it was breaking other processes. This way it was more safe and if it is even faster than great.